type-definitions/ts-tests/exports.ts

```diff
@@ -14,32 +15,98 @@ import {
   Collection,
 } from 'immutable';
 
-List; // $ExpectType typeof List
-Map; // $ExpectType typeof Map
-OrderedMap; // $ExpectType typeof OrderedMap
-OrderedSet; // $ExpectType typeof OrderedSet
-// TODO: Turn on once https://github.com/Microsoft/dtslint/issues/19 is resolved.
-Range; // $ ExpectType (start?: number | undefined, end?: number | undefined, step?: number | undefined) => Indexed<number>
```

Comment:
Notice that few assertions were disabled by adding a space: `// $ ExpectType` instead of `// $ExpectType`. I think now they work as expected.

Reply:
Range [did change in 5.x](https://github.com/immutable-js/immutable-js/blob/5.x/CHANGELOG.md#500-beta5) in that way yes 😉